buildtest/menu/build.py

```diff
@@ -18,7 +18,7 @@
 logger = logging.getLogger(__name__)
 
 
-def discover_buildspecs(buildspec):
+def discover_buildspecs(buildspec, search_path=[]):
```

Comment:
You should generally not set a list as an input argument to a function, it's mutable. Instead this should be set to None, and then `search_path = search_path or []`